guide/advice-for-new-collaborators.rst

```diff
@@ -312,10 +312,26 @@ of the state variables then doing it on demand is more efficient.
 The options here are namelist selectable at runtime and the impact
 on total runtime can be easily measured and compared.
 
-Reuse code when possible
-------------------------
-
-The ``models/template`` directory has files that can be used to start
-porting code to support a new model, but we also recommend looking at
-the existing supported models and reusing code from them if possible.
-Models with similar grid types or vertical coordiates are good candidates.
+New model automation and code reuse
+-----------------------------------
+
+Adding a new model can be done with the script ``models/new_model``.
```

Comment:
The new_model script needs to be run in the models directory, if you run models/new_model

```
$ models/new_model hello oned
oned is not a proper location module! Please try a different location name.
```

Reply:
Ooh, that would be great. Then it would also get rid of the confusion that the threed template works for every 3D component but threed!